images/1.7.7.0-beta.2/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM prestashop/base:7.1-apache
+LABEL maintainer="PrestaShop Core Team <coreteam@prestashop.com>"
+
+ENV PS_VERSION 1.7.7.0-beta.2
```

Comment:
I'm not sure about this one, but in the beta build we don't use the beta suffix but `ENV PS_VERSION 1.7.7.0`
See https://github.com/PrestaShop/docker/blob/master/images/beta/Dockerfile#L9

So it would make sense not to use it here as well

Reply:
It's not a problem, regarding the nightly (ENV PS_VERSION nightly)
And this value is not used in the PrestaShop itself, it's only for Docker scripts.